torch/csrc/monitor/counters.h

```diff
@@ -0,0 +1,186 @@
+#pragma once
+
+#include <mutex>
+#include <unordered_map>
+#include <vector>
+
+#include <c10/util/string_view.h>
+
+namespace torch {
+namespace monitor {
+
+// Aggregation is the list of possible aggregations for Stats.
+// These use bitwise flags so they can be efficiently stored.
+enum Aggregation {
```

Comment:
are we adding variance later?

Reply:
yeah, that's one of the ones that's planned along with histograms